model/context.go

```diff
@@ -92,7 +92,7 @@ func DecodeContext(input interface{}, cfg Config, err error) (*Context, error) {
 	}
 	raw, ok := input.(map[string]interface{})
 	if !ok {
-		return nil, errors.New("Invalid type for fetching Context fields")
+		return nil, errors.New("invalid type for fetching Context fields")
```

Comment:
it would help review a lot if these changes were made in a separate commit, or even a separate PR, as it's not clear initially if it's directly related to the goals fo the changeset.

Reply:
true, sorry for that :ok_hand: